docs/rules/no-import-assign.md

```diff
@@ -18,6 +18,7 @@ mod = 1          // ERROR: 'mod' is readonly.
 named = 2        // ERROR: 'named' is readonly.
 mod_ns.named = 3 // ERROR: the members of 'mod_ns' is readonly.
```

Comment:
```suggestion
mod_ns.named = 3 // ERROR: The members of 'mod_ns' are readonly.
```

Reply:
Let's take the opportunity to fix this as well.

Reply:
👍